packages/oui-datagrid/src/datagrid.html

```diff
@@ -7,6 +7,14 @@
     <oui-datagrid-extra-top class="oui-datagrid-topbar__slot"
         ng-if="$ctrl.hasExtraTopContent">
     </oui-datagrid-extra-top>
+    <button
+        data-ng-if="$ctrl.refreshable"
+        class="oui-datagrid-topbar__refresh oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
+        aria-label="::$ctrl.refreshIconAreaLabel || $ctrl.config.translations.refreshIconAreaLabel"
+        data-ng-click="$ctrl.refreshData(false, false, true, true);"
+    >
+        <span class="oui-icon oui-icon-refresh_extra-thin" aria-hidden="true"></span>
```

Comment:
We can remove `aria-hidden` since it's the only of the button

Reply:
I think it should be hidden from screen readers as aria-label is provided in button and screen reader reads it. It does not need to know about the refresh icon. it's only for people who can see it.

Reply:
You could actually remove it because `aria-label` will overruler everything in the content of the HTML element 🙂